src/domain/payments/price-ratio.ts

```diff
@@ -15,20 +13,30 @@ export const PriceRatio = ({
     return new InvalidZeroAmountPriceRatioInputError()
   }
 
-  const convertFromUsd = (convert: UsdPaymentAmount): BtcPaymentAmount => {
-    const amountAsNumber =
-      (Number(convert.amount) * Number(btc.amount)) / Number(usd.amount)
-    const amount = Math.round(amountAsNumber)
-    const btcAmount = convert.amount === 0n ? 0n : amount === 0 ? 1n : amount
-    return paymentAmountFromSats(toSats(btcAmount))
+  const convertFromUsd = (
+    convert: UsdPaymentAmount,
+  ): BtcPaymentAmount | ValidationError => {
+    if (convert.amount === 0n) {
+      return paymentAmountFromNumber({ amount: 0, currency: WalletCurrency.Btc })
+    }
+
+    const amount = Math.round(
+      (Number(convert.amount) * Number(btc.amount)) / Number(usd.amount),
+    )
+    return paymentAmountFromNumber({ amount: amount || 1, currency: WalletCurrency.Btc })
```

Comment:
why the fallback of 1?

Reply:
It's for this test case: https://github.com/GaloyMoney/galoy/blob/3070b6fb9e45626c9833a28e1cb66e0dd708929d/test/unit/domain/payments/price-ratio.spec.ts#L128